script/find-release.js

```diff
@@ -0,0 +1,39 @@
+if (!process.env.CI) require('dotenv-safe').load()
```

Comment:
This script is called from upload.py in 2-0-x (https://github.com/electron/electron/blob/2-0-x/script/upload.py#L215), but it looks like that change didn't make it back here.